### ISSUE TITLE
[6.x] Skinning in 6.0.3

### ISSUE THREAD

**Issue by Exodius** [TrinityCore/TrinityCore#13878]
I have noticed something and I am making this report as a possible reference for future update when it comes to Skinning. I will try to explain how I came to this observation with all the details so, please don't mind me for writing the wall of text.

I boosted my hunter to LVL90 few days ago and, I decided to get him a Skinning Profession. I learned the skill from [Maris Granger](http://www.wowhead.com/npc=1292), Skinning trainer in Stormwind. I got to Skinning 75 farming boars around [The Maclure Vineyards](http://www.wowhead.com/maps?data=12:417871) because of their fast respawn rate. At 75 I went back to Maris and learned the next tier.

After spending some time farming Wolves and Bears in Duskwood I remembered that I can get [A Guide to Skinning in Draenor](http://www.wowhead.com/item=111351) just like for any other profession when starting to level up in Draenor, totally avoiding the need to visit the trainer as the "Guide" boosts your skill to 700 on it's learning. I Skinned an Elekk in Shadowmoon, got the Guide, learned it and then I thought about the implementation of [Ore Nuggets](http://www.wowhead.com/search?q=ore+nugget) and [Leaves, Stems and Brumbles](http://www.wowhead.com/items=7.9?filter=cr=166;crs=6;crv=0) for herbalism. I could not remember if same thing was implemented for Skinning so, hoping to speed up the process of getting my Skinning to 600 I went to [Sunken Temple](http://www.wowhead.com/zone=1477) because that dungeon met my personal criteria. Dungeon was small, could be completed fast and it was populated 90% by Skinnable creatures (Dragons and Dragonkins).

This is the point where I encountered something interesting... All sources indicated that, by default, creatures of that Level range (52-55) give [Thick](http://www.wowhead.com/item=4304) and [Rugged](http://www.wowhead.com/item=8170) leather when Skinned, however during my first "encounters" with them, all those creatures gave [Medium](http://www.wowhead.com/item=2319) and [Heavy](http://www.wowhead.com/item=4234) Leather.

On first run, 95% of the leather obtained from them was Medium and then they started giving Heavy Leather... The bigger my Skinning skill got, quantity of Medium Leather got lower and Heavy Leather started dropping more and more and then, drop chances got changed again. Medium Leather stopped dropping, Heavy Leather was given to me in 90% of the Skinning attempts and then they started giving me Thick Leather...

Once again, the bigger my Skinning Skill got bigger, Thick Leather was dropping in most cases and they started giving Rugged Leather.

Current Skinning level on my hero is 290. In 70% of the cases, skinning those same Dragonkins give Rugged Leather while 30% gives Thick Leather. I know that Mining and Herbalism underwent some minor overhaul during MoP (you could farm MoP materials at profession skill of 1) however in Draenor it was overhauled more. In regards to other Gathering professions (Mining and Herbalism) you can find documentation about the changes (Nuggets and "Leaves" linked above) however I could not find anything related to Skinning so I am making this report to see did anyone else encounter this with Skinning.

**Comment by MrSmite** [comment]
I think this also happens in the 335 branch. I sometimes get higher tier leathers from wolves around Elwynn.

**Comment by Exodius** [comment]
Yeah, but on this scale? According to WoWhead and some other Sources / Guides available, those creatures in Sunken Temple should drop only Thick and Rugged Leather. However, they started giving Medium and Heavy first and then (when my skill got to appropriate level for obtaining them) they started giving Thick and Rugged leather.

Although my Skill Level is 290 at the moment, I will have to drop the skill entirely, re-learn it and then start again from 1 just so I can test it properly. I'll see what I can do. I'll post the findings when I do this.

**Comment by danlapps** [comment]
Do you need a sniff of the zone skinned from a lvl 700 Skinner or is it useless for your tests ?

**Comment by MrSmite** [comment]
> Although my Skill Level is 290 at the moment, I will have to drop the skill entirely, re-learn it and then start again from 1 just so I can test it properly. I'll see what I can do. I'll post the findings when I do this.

Well not really, you can just use __.set skill 393 250 700__ and then increment it. Don't forget, those [Dragonkin](http://www.wowhead.com/npc=5277#skinning) are supposed to have a chance to drop Dragonscale and Worn Dragonscale when skinned, not just leather.

**Comment by Exodius** [comment]
Thanks for the suggestion, @MrSmite but I can't do that.
My report is an observation made while playing on Retail... :smile: Not on a Emu set up on my PC...
This is something that happens on retail and this report was made as a feedback topic for gathering information.

And as for Dragonscale, so far they haven't dropped any.

**Comment by Exodius** [comment]
It is as I suspected. Leather drop changes along the same path as your Skinning Skill...

Dropped Skinning and learned it again. Went to Draenor at skill LVL1 to obtain [the Guide](http://www.wowhead.com/item=111351) and in process gained 1 skill and 2 x [Raw Beast Hide Scraps](http://www.wowhead.com/item=110610) which I will exclude from the results.

Skill 2 - 75
23 x [Ruined Leather Scraps](http://www.wowhead.com/item=2934)
71 x [Light Leather](http://www.wowhead.com/item=2318)
07 x [Light Hide](http://www.wowhead.com/item=783)
NOTE: At first dozen skins, drop was in favour of Ruined Leather Scraps however, around skill 40,  they stopped dropping. 0 x Ruined Leather Scraps after Skill 40.

Skill 76 - 150
39 x [Light Leather](http://www.wowhead.com/item=2318)
58 x [Medium Leather](http://www.wowhead.com/item=2319)
06 x [Heavy Leather](http://www.wowhead.com/item=4234)
03 x [Light Hide](http://www.wowhead.com/item=783)
04 x [Medium Hide](http://www.wowhead.com/item=4232)
01 x [Heavy Hide](http://www.wowhead.com/item=4235)
NOTE: First [Medium Leather](http://www.wowhead.com/item=2319) was obtained on Skill 81.
NOTE: [Light Leather](http://www.wowhead.com/item=2318) stopped dropping at Skill 120.
NOTE: First [Heavy Leather](http://www.wowhead.com/item=4234) was obtained on Skill 140.

Skill 151 - 225
09 x [Medium Leather](http://www.wowhead.com/item=2319)
56 x [Heavy Leather](http://www.wowhead.com/item=4234)
38 x [Thick Leather](http://www.wowhead.com/item=4304)
02 x [Medium Hide](http://www.wowhead.com/item=4232)
05 x [Heavy Hide](http://www.wowhead.com/item=4235)
NOTE: [Medium Leather](http://www.wowhead.com/item=2319) stopped dropping at Skill 170.
NOTE: First [Thick Leather](http://www.wowhead.com/item=4304) was obtained on Skill 190.
NOTE: [Heavy Leather](http://www.wowhead.com/item=4234) stopped dropping at Skill 214.

Skill 226 - 300
At this point, the drop percentage between Thick and Rugged Leather was more or less 50/50.
58 x [Thick Leather](http://www.wowhead.com/item=4304)
49 x [Rugged Leather](http://www.wowhead.com/item=8170)
02 x [Thick Hide](http://www.wowhead.com/item=8169)
01 x [Rugged Hide](http://www.wowhead.com/item=8171)
NOTE: First [Rugged Leather](http://www.wowhead.com/item=8170) was obtained on Skill 232.

**Comment by Nyr97** [comment]
Skinning has changed entirely, especially recently due to loot changes and difficulties. Is this valid?

**Comment by Exodius** [comment]
Skinning test results provided are done on a character that has been long deleted. I tend to do that.

I've edited the main post with a "8 year after edit", hopefully clarifying things more.

I'd have to do a new test in order to confirm/debunk the theory since professions have, indeed, changed since the release of DF.

[Patch 8.0.1](https://wowpedia.fandom.com/wiki/Patch_8.0.1) (2018-07-17): Each expansion now has a separate profession skill bar, with smaller caps for each expansion, instead of one gigantic profession skill bar for all expansions put together.

Back in WoD, you could get [the Guide](https://www.wowhead.com/item=111351) by having a profession just learned and then skinning the first creature you encounter. It unlocked the profession to the max, eliminating the need to visit the Trainer and learn the next tier so you can increase the points.

The point of my experiment was that, I got into an area that has:
- Same type of creatures (Skinnable dragons and dragonkin)
- Constant level of the creatures that is not variable.
- Prove you can obtain wide variety of materials that would, usually, require you to go from zone to zone, in one location.

Even though profession's skill points are not as gradual now as it was back in WoD, but segmented by their respective expansions, and my findings here are related to first 300 points of the Classic, I'd say this still holds but, without actual testing, I can't say for sure.
